ci/java-mvn/src/test/java/io/ryandeford/samples/data/SampleResultDataTest.java

```diff
@@ -0,0 +1,39 @@
+package io.ryandeford.samples.data;
+
+import io.ryandeford.samples.test.SampleTestConstants;
+import io.ryandeford.samples.utils.SampleConstants;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class SampleResultDataTest {
+
+    @Test
+    void createSampleResultDataDefaultConstructor() {
+        SampleResultData data = new SampleResultData();
+        assertEquals(SampleConstants.DEFAULT_SAMPLE_RESULT_MESSAGE, data.getMessage());
+    }
+
+    @Test
+    void createSampleResultDataMessageConstructor() {
+        String message = "Custom Sample Message";
+        SampleResultData data = new SampleResultData(message);
+        assertEquals(message, data.getMessage());
+    }
+
+    @Test
+    void createSampleResultDataBuilderDefault() {
+        SampleResultData data = SampleResultData.builder().build();
+        assertEquals(SampleConstants.DEFAULT_SAMPLE_RESULT_MESSAGE, data.getMessage());
+    }
+
+    @Test
+    void createSampleResultDataBuilderCustomMessage() {
+        SampleResultData data = SampleResultData.builder()
+                .message(SampleTestConstants.CUSTOM_SAMPLE_RESULT_MESSAGE)
+                .build();
+        assertEquals(SampleTestConstants.CUSTOM_SAMPLE_RESULT_MESSAGE, data.getMessage());
+    }
+
+    // Note: There are all kinds of tests we could create here, but this is just a sample project
```

Comment:
If this were a real-life production codebase, we'd create a lot more tests around the Lombok-generated code features, plus we'd beef up the other basic tests, but this is just for demonstration purposes.